### PR TITLE
Ra 133 fe display detailed information

### PR DESCRIPTION
**Pull request by AntonioMorales2**

### CODE REVIEW

File: frontend/src/ui/section/event-detail/EventDetail.tsx [hunk 1]
```diff
@@ -1,62 +1,109 @@
...
-  } = eventMock;
-
-  // TODO: When backend is ready check if event exist or not and modify this condition
+  const [eventDetails, setEventDetails] = useState<DetailedEvent>();
+  const [hasError, setHasError] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+
+  const eventRepository = createEventRepository();
+
+  const emojiConvertor = new EmojiConvertor.EmojiConvertor();
```
Comment (pedro-dominguez-rindus) [inline frontend/src/ui/section/event-detail/EventDetail.tsx:26]: Now its not necessary, but maybe could be good idea create a helper or hook for that in the future if we think to reuse it in more places

File: frontend/src/ui/section/event-detail/EventDetail.tsx [hunk 2]
```diff
@@ -1,62 +1,109 @@
...
-          weekday={weekday}
-          colour="#000000"
-        />
+    <Section
+      className="eventDetail"
+      refresh={() => load(id)}
+      isLoading={isLoading}
+      shouldRefresh={hasError}
+    >
+      {eventDetails ? (
```
Comment (pedro-dominguez-rindus) [inline frontend/src/ui/section/event-detail/EventDetail.tsx:69]: this condition is not good

File: frontend/src/ui/section/event-detail/EventDetail.tsx [hunk 3]
```diff
@@ -1,62 +1,109 @@
...
+                  eventDetails?.description,
+                ),
+              }}
+            ></div>
+          </div>
+          <div>
+            <IconWithText
+              icon={<img alt="Calendar" src={calendarClockImage} />}
+            >
+              {eventDetails.time}
```
Comment (pedro-dominguez-rindus) [inline frontend/src/ui/section/event-detail/EventDetail.tsx:93]: Be solid, all access to the eventDetails properties are with ?. like   title={eventDetails?.summary?.name}
            month={eventDetails?.summary?.month}
            day={eventDetails?.summary?.day}
            
            but for the rest you dont use Nullish coalescing operator.
            
           Apply the same logic without  Nullish coalescing operator for all properties

File: frontend/src/ui/section/event-detail/EventDetail.tsx (hunk 1 above)
Comment (AntonioMorales2) [reply frontend/src/ui/section/event-detail/EventDetail.tsx:26]: Correct, I will work on it with the test. Leaving a comment in the code

File: frontend/src/ui/section/event-detail/EventDetail.tsx (hunk 3 above)
Comment (AntonioMorales2) [reply frontend/src/ui/section/event-detail/EventDetail.tsx:93]: done!

File: frontend/src/ui/section/event-detail/EventDetail.tsx (hunk 2 above)
Comment (AntonioMorales2) [reply frontend/src/ui/section/event-detail/EventDetail.tsx:69]: done

File: frontend/src/mocks/event.ts [hunk 4]
```diff
@@ -1,35 +1,36 @@
...
+  location: {
+    url: string;
+    name: string;
+    address: string;
+  };
+  time: string;
+  petsAllowed: true;
+}
+
+export const eventMock: Event = {
```
[the remark this answers is not in the document]
Comment (AntonioMorales2) [reply frontend/src/mocks/event.ts:19]: done

**Pull request merged**